torch_geometric/nn/conv/lg_conv.py

```diff
@@ -0,0 +1,48 @@
+from torch_geometric.typing import Adj, Size
+
+from torch import Tensor, ones
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.utils import degree
+
+
+class LGConv(MessagePassing):
+    r"""The Light Graph Convolution (LGC) operator from the `"LightGCN:
+    Simplifying and Powering Graph Convolution Network for Recommendation"
+    <https://arxiv.org/abs/2002.02126>`_ paper.
```

Comment:
Since this implementation adds additional functionalities on top of the vanilla LightGCN, I think it's worth mentioning what those functionalities are and how to use them here (eg. switching normalization on/off), so that the users are aware.

Reply:
Agree :) Would you mind editing the documentation?

Reply:
Done :)